alicja/config/proto/interface.proto

```diff
@@ -0,0 +1,64 @@
+// This proto defines the Hermes - Cloud Prober interface config
+
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+
+// HermesProber provides methods (PutFile, GetFile, DeleteFile) to communicate instructions to Cloudprober regarding file operations 
+
+service HermesProber {
+  // Create and upload a file to the storage system
+  rpc PutFile(PutFileRequest) returns PutFileResponse;
+  
+  // Retrieve a file from the storage system
+  rpc GetFile(GetFileRequest) returns GetFileResponse;
+  
+  // Delete a file from the storage system
+  rpc DeleteFile(DeleteFileRequest) returns DeleteFileResponse;
+}
+
+// Left for future PR
```

Comment:
Use the TODO tag and your github username or the issue number.
```suggestion
// TODO(#<Issue Number>) Complete exit code enum in interface.proto.
```

Reply:
```suggestion
// TODO (#31) Complete ExitCode enum in interface.proto
```